app/code/Magento/Ui/Component/ExportButton.php

```diff
@@ -54,16 +54,39 @@ public function getComponentName()
      */
     public function prepare()
     {
+        $context = $this->getContext();
         $config = $this->getData('config');
         if (isset($config['options'])) {
             $options = [];
             foreach ($config['options'] as $option) {
-                $option['url'] = $this->urlBuilder->getUrl($option['url']);
+                $additionalParams = $this->getAdditionalParams($config, $context);
+                $option['url'] = $this->urlBuilder->getUrl($option['url'], $additionalParams);
                 $options[] = $option;
             }
             $config['options'] = $options;
             $this->setData('config', $config);
         }
         parent::prepare();
     }
+
+    /**
+     * Get export button additional parameters
+     *
+     * @param array $config
+     * @param ContextInterface $context
+     * @return array
+     */
+    protected function getAdditionalParams($config, $context)
```

Comment:
Hi @rogyar , due to Magento backward compatibility we can't introduce new public and protected methods. Please change it to private. 
[Backward compatible development](http://devdocs.magento.com/guides/v2.1/contributor-guide/backward-compatible-development/index.html)